config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
 
+  resources :campaigns, only: [:new, :create, :edit, :update, :index, :show]
```

Comment:
If you're going to use most of the seven crud actions, use `resources :campaigns, except: :delete`